internal/cli/config/config.go

```diff
@@ -54,6 +63,8 @@ type Config struct {
 	RefreshToken     string                      `json:"refresh_token,omitempty"`
 	filePath         string
 	apiClient        api.ClientInterface
+	// Dac wraps the configuration related to Dashboard-as-Code
+	Dac Dac `json:"dac,omitempty" yaml:"dac,omitempty"`
```

Comment:
I would keep the exposed attribute at the beginning and the others at the end of the struct.
Also no need to specify a tag for yaml, the struct is only marshalled in JSON.